dpnp/linalg/dpnp_utils_linalg.py

```diff
@@ -2532,13 +2534,15 @@ def dpnp_qr(a, mode="reduced"):
             a_t,
             shape=(m, m),
             dtype=res_type,
+            order="C",
```

Comment:
I guess the PR description is not fully correct, because there is `_geqrf` and `_geqrf_batch` functions but not `getrf`.

Reply:
Yeah, I made a mistake.
Updated the PR description